### PR TITLE
changed subscription on MainPageView with the functionality of navigat…

### PR DESCRIPTION
**Pull request by DenysIvashchenko**
…ion bar page tracking

### CODE REVIEW

**Review by litvinets — APPROVED** [review]

File: src/app/shell/personal-cabinet/personal-cabinet.component.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   background: #FFFFFF;
   box-shadow: 0px 6px 16px rgba(0, 0, 0, 0.08);
   padding: 24px;
-  margin-top:80px;
+  // margin-top:80px;
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/personal-cabinet.component.scss:5]: ?

**Review by dmitryy90 — APPROVED** [review]

File: src/app/shell/personal-cabinet/personal-cabinet.component.scss (hunk 1 above)
Comment (DenysIvashchenko) [reply src/app/shell/personal-cabinet/personal-cabinet.component.scss:5]: correct
it applies the correct display of "КАБІНЕТ КОРИСТУВАЧА"

Comment (dmitryy90) [reply src/app/shell/personal-cabinet/personal-cabinet.component.scss:5]: do we need commented styles?

Comment (DenysIvashchenko) [reply src/app/shell/personal-cabinet/personal-cabinet.component.scss:5]: I forgot to delete  :)

**Pull request merged**